src/core/p3m.cpp

```diff
@@ -1207,12 +1207,7 @@ template <int cao> void calc_influence_function_force() {
                  Utils::sqr(p3m.d_op[RY][n[KY]] / box_l[RY]) +
                  Utils::sqr(p3m.d_op[RZ][n[KZ]] / box_l[RZ]);
 
-          if(fak2 == 0) {
-            fak3 = 0;
-          }
-          else
-            fak3 = fak1 / (fak2 * Utils::sqr(denominator));
-
```

Comment:
Looks like you made an error resolving a merge conflict. The check for zero was intentionally added in #2130.